src/components/personkort/PersonkortEnhet.tsx

```diff
@@ -35,14 +35,19 @@ const PersonkortEnhet = () => {
         <AppSpinner />
       ) : behandlendeenhet ? (
         <PersonkortElement
-          tittel={behandlendeenhet.oppfolgingsenhet.navn}
+          tittel={
+            behandlendeenhet.oppfolgingsenhet?.navn ??
+            behandlendeenhet.geografiskEnhet?.navn
```

Comment:
Ser ikke helt at `?` er nødvendig her med mindre geografisk enhet kan være `null`.
```suggestion
            behandlendeenhet.geografiskEnhet.navn
```